lua/zones/dungeon00_level01/init.lua

```diff
@@ -0,0 +1,109 @@
+
+obelisk = WorldEntity.new("world.checkpoints.TutorialCheckpointEntity")
+currentZone:spawn(obelisk, Vector3.new(757, 289, 35), 346)--X: 143048 Y: 147450 Z: 2559 Rot: 361.16
+--portal = ZonePortal.new("unk1", "unk2")
+portal = ZonePortal.new("unk1", "unk2")
+portal:target("tutorial")
+portal:width(75)
+portal:height(75)
+portal:unk4(0xFFFFFFFF)
+currentZone:spawn(portal, Vector3.new(641, -557, 35), 180)--X: 143116 Y: 246097 Z: 2559 Rot: 2.59
+
+portal = ZonePortal.new("unk1", "unk2")
+portal:target("dungeon00_level02")
+portal:width(75)
+portal:height(75)
+portal:unk4(0xFFFFFFFF)
+
+currentZone:spawn(portal, Vector3.new(560, 962, 35), 180)--X: 143336 Y: 246261 Z: 2559 Rot: 0.76
+
+
+
+
+
+--ZonePortal_oneway = WorldEntity.new("misc.ZonePortal_oneway")
+--currentZone:spawn(ZonePortal_oneway, Vector3.new(757, 289, 40), 346)
+
+
+
+    --currentZone:spawn(vendor, Vector3.new(860, 490 , 40), 180)
+
+
+
```

Comment:
There are a lot of unused comments and duplicate comments from other scripts, it would be much easier to read if comments that are not useful are removed please.